tests/python/pants_test/option/test_options.py

```diff
@@ -1398,6 +1383,14 @@ def test_double_registration(self):
     options.register(GLOBAL_SCOPE, '--foo-bar')
     self.assertRaises(OptionAlreadyRegistered, lambda: options.register(GLOBAL_SCOPE, '--foo-bar'))
 
+  def test_serializability(self):
+    # We serialize options to JSON e.g., when uploading stats.
+    # This test spot-checks that enum types can be serialized.
+    options = self._parse('./pants enum-opt --some-enum=another-value')
+    # Note that for some reason CoercingEncoder doesn't do the right thing for enum-valued opts
```

Comment:
How does "... CoercingEncoder doesn't do the right thing ..." map to the fact that the next line uses it and apparently does not blow up? Is the note intended to be a TODO or is it just worded funny or am I just reading it funny?

Reply:
Oh wait oops! I copy and pasted from Benjy’s failing test naively without closely reading. Stale comment.